### PR TITLE
Added the HTML backbone of the app

### PR DESCRIPTION
**Pull request by tzavidas**
Added the backbone of the app and structured it. Changes only to the HTML file!

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,56 @@
...
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 10px;
+    grid-row-gap: 10px;
+    padding: 10px 10px 0;
+}
+
+.card
+{
+    height: 200px;
+}
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/style.css]: No newline at end of file?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,85 @@
...
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/index.html:7]: Where is this file?

**Review by maxwbuckley — COMMENTED** [review]
How can I see this? Can you share a screenshot?

**Comment by tzavidas** [comment]
Reverted the CSS changes. Only intended to add the backbone on this PR. I will make a separate one for the CSS styling

**Review by maxwbuckley — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (tzavidas) [reply portfolio/src/main/webapp/index.html:7]: The files are:
script.js: https://github.com/tzavidas/step-tutorial/blob/master/portfolio/src/main/webapp/script.js
style.css: https://github.com/tzavidas/step-tutorial/blob/master/portfolio/src/main/webapp/style.css

**Pull request merged**